File: app/src/main/java/org/dhis2/usescases/main/program/ProgramComponent.java
package org.dhis2.usescases.main.program;

import org.dhis2.data.dagger.PerFragment;

import dagger.Subcomponent;

/**
 * QUADRAM. Created by ppajuelo on 07/02/2018.
 */
@PerFragment
@Subcomponent(modules = ProgramModule.class)
public interface ProgramComponent {
    void inject(ProgramFragment programFragment);
}

File: app/src/main/java/org/dhis2/usescases/main/program/ProgramComponent.kt
package org.dhis2.usescases.main.program

import dagger.Subcomponent
import org.dhis2.commons.di.dagger.PerFragment

@PerFragment
@Subcomponent(modules = [ProgramModule::class])
interface ProgramComponent {
    fun inject(programFragment: ProgramFragment)
}
